wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -36,7 +36,6 @@ section {
 /* ___ Homepage ___ */
 
 .site-header.home .site-title {
-	font-size: 3.10rem;
```

Comment:
Removing this means that the font-size increases on larger screens due to the legacy styles not being overridden. Could you try instead scoping this rule to only screens larger than the mobile breakpoint?

```
.site-header.home .site-title {
	font-weight: 400;
	line-height: normal;
	
	@media only screen and (min-width: $breakpoint-small) {
		font-size: 3.10rem;
	}
}
```

Reply:
@adamwoodnz Sorry for the delay, I have updated the CSS now.